Makefile

```diff
@@ -362,6 +363,10 @@ EXE = .exe
 
 else ifneq ($(CONFIG),none)
 $(error Invalid CONFIG setting '$(CONFIG)'. Valid values: clang, gcc, emcc, mxe, msys2-32, msys2-64)
+else
+$(info Using provided C++ compiler, '$(CXX)'.)
```

Comment:
Is this going to be printed on every `make` invocation? That's distracting.

Reply:
Good point (although it would "only" be every `make` invocation without a value for `CONFIG`).  I think it's still useful for debug purposes but it now requires `WHICH_CXX` to be defined so it is opt-in.